src/storage/test/QueryBoundTest.cpp

```diff
@@ -272,6 +339,78 @@ TEST(QueryBoundTest,  GenBucketsTest) {
     }
 }
 
+TEST(QueryBoundTest, FilterTest_TagAndEdgeFilter) {
+    fs::TempDir rootPath("/tmp/QueryBoundTest.XXXXXX");
+    LOG(INFO) << "Prepare meta...";
+    std::unique_ptr<kvstore::KVStore> kv(TestUtils::initKV(rootPath.path()));
+    auto schemaMan = TestUtils::mockSchemaMan();;
+    mockData(kv.get());
+
+    LOG(INFO) << "Build filter...";
+    auto* tag = new std::string("3001");
+    auto* prop = new std::string("tag_3001_col_0");
+    auto* srcExp = new SourcePropertyExpression(tag, prop);
+    auto* priExp = new PrimaryExpression(20 + 3001L);
+    auto* left = new RelationalExpression(srcExp,
+                                          RelationalExpression::Operator::GE,
+                                          priExp);
+    auto* edgeProp = new std::string("col_0");
+    auto* alias = new std::string("e101");
+    auto* edgeExp = new EdgePropertyExpression(alias, edgeProp);
+    auto* priExp2 = new PrimaryExpression(10007L);
+    auto* right = new RelationalExpression(edgeExp,
+                                           RelationalExpression::Operator::GE,
```

Comment:
use LE

Reply:
If LE,  all edges in UT will pass the filter.